test/Microsoft.AspNetCore.Server.Kestrel.Performance/project.json

```diff
@@ -0,0 +1,42 @@
+﻿{
+  "version": "1.0.0-*",
+  "dependencies": {
+    "BenchmarkDotNet": "0.10.0",
+    "Microsoft.AspNetCore.Server.Kestrel": "1.2.0-*"
+  },
+  "frameworks": {
+    "netcoreapp1.0": {
+      "dependencies": {
+        "Microsoft.NETCore.App": {
+          "version": "1.0.1-*",
+          "type": "platform"
+        }
+      }
+    }
+  },
+  "buildOptions": {
+    "emitEntryPoint": true,
+    "compile": {
+      "include": [
+        "../shared/SocketInputExtensions.cs",
```

Comment:
I just noticed you aren't using `"../shared/**/*.cs"` like the other test projects. Any reason for that?